### ISSUE TITLE
[BUG] Can't run tests more than once with new Test Runner

### ISSUE THREAD

**Issue by webreidi** [microsoft/vscode-dotnettools#1474]
### Describe the Issue

I created a test project using MSTest and the new Test Runner. When I run the tests the first time, they run great. But if I try to run the tests a second time, it won't run, complaining of a build error. The output for build says that the exe for the test project is being used by another process. 

I tried closing and reopening VS Code, and cleaning the project. Still same problem. The only solution I've found is to delete close VS Code, delete the net8.0 folder and reopen and then I can re-run the tests.

C:\Program Files\dotnet\sdk\9.0.100-rc.1.24452.12\Microsoft.Common.CurrentVersion.targets(5322,5): warning MSB3026: Could not copy "C:\Users\webreidi\source\repos\codel-cloud-native\FrontEnd.Tests\obj\Debug\net8.0\apphost.exe" to "bin\Debug\net8.0\FrontEnd.Tests.exe". Beginning retry 10 in 1000ms. The process cannot access the file 'C:\Users\webreidi\source\repos\codel-cloud-native\FrontEnd.Tests\bin\Debug\net8.0\FrontEnd.Tests.exe' because it is being used by another process. The file is locked by: "FrontEnd.Tests (5112)" [C:\Users\webreidi\source\repos\codel-cloud-native\FrontEnd.Tests\FrontEnd.Tests.csproj]

**Comment by webreidi** [comment]
Actually just checked and I'm targeting .NET8, but other than that basically the same as yours. I don't have the ItemGroup section because those are included in the SDK.

I created a .NET Aspire project and then added an MSTest project to it for my unit tests.

**Comment by AbhitejJohn** [comment]
@ocallesp : Didn't we fix this scenario and insert a fix in last week?

**Comment by AbhitejJohn** [comment]
@ocallesp : Checking back in on this. Have we addressed this already?

**Comment by webreidi** [comment]
Yes,  verified that I can run tests multiple times without reloading now!
